frame/transaction-payment/asset-tx-payment/src/tests.rs

```diff
@@ -152,6 +152,10 @@ impl pallet_transaction_payment::Config for Runtime {
 	type OperationalFeeMultiplier = ConstU8<5>;
 }
 
+parameter_types! {
+	pub const RemoveKeysLimit: u32 = 1000;
+}
+
```

Comment:
Have a look at the alliances runtime - you can see that instead of using `parameter_types` we can use `ConstU32<1000>` - see https://github.com/paritytech/cumulus/blob/e23d581b768d0e6b8e1593408f12274c0de652f4/parachains/runtimes/collectives/collectives-polkadot/src/lib.rs#L185 . I think we can do the same in pallets.

Reply:
I see. That's definitely more elegant/self contained. I'll do that.